### DISCUSSION TITLE
Welcome to Lenovo-Ideapad-320-15ISK-14ISK-Laptop-Hackintosh Discussions!

### DISCUSSION THREAD

**Discussion by gajjartejas** [gajjartejas/Lenovo-Ideapad-320-15ISK-14ISK-Laptop-Hackintosh#40] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OriginsCodex** [comment]
https://www.youtube.com/watch?v=Sekf0yn9FVw

Look at this ac 3165 working sorry about my english im brazilian learned by myself lol

**Reply by gajjartejas** [reply]
Yes, WiFi card is working.
